s3prl/upstream/mos_prediction/hubconf.py

```diff
@@ -0,0 +1,104 @@
+import os
+
+from s3prl.utility.download import _urls_to_filepaths
+from .expert import UpstreamExpert as _UpstreamExpert
+
+
+
+def mos_wav2vec2_local(ckpt, *args, **kwargs):
+    """
+        The model from local ckpt
+            ckpt (str): PATH
+    """
+    assert os.path.isfile(ckpt)
+    kwargs['upstream'] = 'wav2vec2'
+    return _UpstreamExpert(ckpt, *args, **kwargs)
+
+def mos_wav2vec2_gdriveid(ckpt, refresh=False, *args, **kwargs):
+    """
+        The model from google drive id
+            ckpt (str): The unique id in the google drive share link
+            refresh (bool): whether to download ckpt/config again if existed
+    """
+    return mos_wav2vec2_local(_gdriveids_to_filepaths(ckpt, refresh=refresh), *args, **kwargs)
```

Comment:
We will not consider g-drive download anymore. G-drive doesn't allow too many scripted downloading. So you can remove all the upstream entries related to g-drive.